main.go

```diff
@@ -25,89 +22,18 @@ import (
 )
 
 const (
-	connectionTimeoutEnv   string = "RYUK_CONNECTION_TIMEOUT"
-	portEnv                string = "RYUK_PORT"
-	reconnectionTimeoutEnv string = "RYUK_RECONNECTION_TIMEOUT"
-	ryukLabel              string = "org.testcontainers.ryuk"
-	verboseEnv             string = "RYUK_VERBOSE"
+	ryukLabel string = "org.testcontainers.ryuk"
 )
 
 var (
-	port                int
+	port                uint16
 	connectionTimeout   time.Duration
 	reconnectionTimeout time.Duration
 	verbose             bool
 )
 
-type config struct {
-	Port                int
-	ConnectionTimeout   time.Duration
-	ReconnectionTimeout time.Duration
-	Verbose             bool
-}
-
-// newConfig parses command line flags and returns a parsed config. config.timeout
```

Comment:
We are losing the command flags support in this PR, right? Is this intended?

Reply:
Have you considered using https://github.com/ardanlabs/conf/blob/master/doc.go?

Reply:
I couldn't see how that could have ever been used so didn't see any point in keeping. Correct me if I'm wrong?

Reply:
Yeah, this is a valid concern. @eddumelendez @kiview what do you think? @stevenh raises a valid concern here about removing the flags support: are the Ryuk flags used anywhere else? If used in any other project, external to testcontainers, are we fine removing that support, keeping the env vars only?

From my point of view, the simplest, the best, so removing that support would reduce the codebase.

Reply:
I don't think any issue removing the flags

Reply:
As long as we don't break integration with other Testcontainers implementations, it is fine 👍

Reply:
If they are using the container, all is good.